### ISSUE TITLE
glance_image can't set owner to non-member tenant

### ISSUE THREAD

**Issue by peterjenkins1** [ansible/ansible-modules-core#821]
The `owner` parameter of the `glance_image` module only works when 'login_tenant_name' is the same. You can't set `login_tenant_name` to a tenant that you aren't a member because you get a permissions error.

The glance command line tools and v2 api don't have this issue.

**Comment by ansibot** [comment]
@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by rcarrillocruz** [comment]
Hi @peterjenkins1 

glance_image is considered deprecated, you should use os_image instead.

Please try that module your playbook and report back if you have the same issues.

Thanks

**Comment by ansibot** [comment]
@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by Shrews** [comment]
glance_image is no longer maintained.

wontfix
